components/Form.js

```diff
@@ -0,0 +1,87 @@
+import { styled } from "styled-components";
+import PrimaryLink from "./PrimaryLink";
+
+export default function Form({ currentTrips, setCurrentTrips }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    const newTripObject = {
+      tripName: data.name,
+      tripDestination: data.destination,
+    };
+    setCurrentTrips([...currentTrips, newTripObject]);
+    event.target.reset();
+  }
+  return (
+    <StyledForm onSubmit={handleSubmit}>
+      <label htmlFor="name">
+        Name of your trip:
+        <StyledInput
+          type="text"
+          name="name"
+          placeholder="Sprachreise"
```

Comment:
```suggestion
          placeholder="e.g. culinary trip, language study travel"
```
damit es eindeutlicher ein placeholder ist und nicht "vorausgefüllt" aussieht :) und am besten alles in einer Sprache!

Reply:
like me in this review 🤣🙈